setup.py

```diff
@@ -25,27 +19,25 @@
       classifiers=['Development Status :: 6 - Mature',
                    'Environment :: Console',
                    'Intended Audience :: Science/Research',
-                   'License :: OSI Approved :: GNU General Public License v2 (GPLv2)',
+                   'License :: OSI Approved :: GNU General Public License v2+ (GPLv2+)',
```

Comment:
```suggestion
                   'License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)',
```
See: https://github.com/pypa/trove-classifiers/blob/main/src/trove_classifiers/__init__.py#L276